data/import/glossary.csv

```diff
@@ -3129,3 +3129,7 @@ product_packaging_unit_node_summary,(= %summary%),en_US
 product_packaging_unit_node_summary,(= %summary%),de_DE
 product_packaging_unit_node_amount_summary,= %summary%,en_US
 product_packaging_unit_node_amount_summary,= %summary%,de_DE
+merchant_switcher.message,"Wechseln von %currentMerchant% zu %newMerchant%? Aufgrund unterschiedlicher Verfügbarkeit können ggf. Nicht alle Produkte in Warenkorb übernommen werden.",de_DE
```

Comment:
typo 
- "können ggf. Nicht alle Produkte" => nicht lowercase
- "in Warenkorb" => "in den Warenkorb" ?

Reply:
können ggf. nicht alle Produkte --> yes, should be small one.
second one should be fine I beileve? Or does it sound weird without the article?

Reply:
This typo will be fixed in https://spryker.atlassian.net/browse/MP-2064 
Thanks, @dereuromark.